js/bitget3.js

```diff
@@ -0,0 +1,2899 @@
+        //         "id":"671701716584665088",
+        //         "price":"150.000000000000",
+        //         "source":"接口",
+        //         "state":"canceled",
+        //         "symbol":"eth_usdt",
+        //         "type":"buy-limit"
+        //     }
+        //
+        //     swap
+        //
+        //     {
+        //         "symbol":"cmt_ethusdt",
+        //         "size":"1",
+        //         "timestamp":"1595885546770",
+        //         "client_oid":"f3aa81d6-9a4c-4eab-bebe-ebc19da21cf2",
+        //         "createTime":"1595885521200",
+        //         "filled_qty":"0",
+        //         "fee":"0.00000000",
+        //         "order_id":"671758053112020913",
+        //         "price":"150.00",
+        //         "price_avg":"0.00",
+        //         "status":"0",
+        //         "type":"1",
+        //         "order_type":"0",
+        //         "totalProfits":null
+        //     }
+        //
+        let id = this.safeString (order, 'order_id');
+        id = this.safeString2 (order, 'id', 'data', id);
+        const timestamp = this.safeInteger2 (order, 'created_at', 'createTime');
+        let type = this.safeString (order, 'type');
+        const side = this.parseOrderSide (type);
+        type = this.parseOrderType (type);
+        // if ((side !== 'buy') && (side !== 'sell')) {
+        //     side = this.parseOrderSide (type);
+        // }
+        // if ((type !== 'limit') && (type !== 'market')) {
+        //     if ('pnl' in order) {
+        //         type = 'futures';
+        //     } else {
+        //         type = 'swap';
+        //     }
+        // }
+        let symbol = undefined;
+        const marketId = this.safeString (order, 'symbol');
+        if (marketId !== undefined) {
+            if (marketId in this.markets_by_id) {
+                market = this.markets_by_id[marketId];
+            } else {
+                symbol = marketId.toUpperCase ();
+            }
+        }
+        if ((symbol === undefined) && (market !== undefined)) {
+            symbol = market['symbol'];
+        }
+        const amount = this.safeString2 (order, 'amount', 'size');
+        const filled = this.safeString2 (order, 'filled_amount', 'filled_qty');
+        const cost = this.safeString (order, 'filled_cash_amount');
+        const price = this.safeString (order, 'price');
+        const average = this.safeString (order, 'price_avg');
+        const status = this.parseOrderStatus (this.safeString2 (order, 'state', 'status'));
+        const feeCost = this.safeNumber2 (order, 'filled_fees', 'fee');
+        let fee = undefined;
+        if (feeCost !== undefined) {
+            const feeCurrency = undefined;
+            fee = {
+                'cost': feeCost,
+                'currency': feeCurrency,
+            };
+        }
+        const clientOrderId = this.safeString (order, 'client_oid');
+        return this.safeOrder2 ({
```

Comment:
`safeOrder2` does not exist anymore, `safeOrder` should be used instead

Reply:
Fixed in PR #11907